jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/SliderRange.java

```diff
@@ -30,42 +30,44 @@ private void setAttributes(int thumbIndex, int value, String newThumbStyle) {
   }
 
   private void reflectionSetValue(int thumbIndex, String newThumbStyle,
-                                  String newStyle, String trackStyle,int value, int left, int width) {
+                                  String newStyle, int value, int Left, int width) {
```

Comment:
Please use "left" variant of variable from small letter

Reply:
done